### PR TITLE
Check if subscription is ready

### LINKED ISSUE #2320: Organization view: Click on any API shows user info. text momentarily
# Steps 
1.Visit https://apinf.io  v 0.41.0 
2.Login as admin/owner  
3.Click on Organizations
4.Go to any organization
5.Select and click on any API 

# Findings  
- shows info message "This Organization doesn't have any connected APIs " momentarily 
- Also reproducible on nightly V 0.41.0

### LINKED ISSUE #2486: Organization view: Sort APIs flashes user message momentarily
# Steps  
1.Visit https://staging.apinf.io   v 0.43.0 
2.Login as admin 
3.Go to Organizations 
4.Select and click on organization (eg. Test2_organization)   
5.Go to APIs 
6.Sort APIs (eg. by Bookmarks/Newest created)  

# Findings 
Sort APIs flashes user message " This organization doesn't have any connected APIs........"  momentarily

### PR DESCRIPTION
**Pull request by 55**
Closes #2486 
Closes #2320

### CODE REVIEW

**Comment by 55** [comment]
Approach with Template.subscriptionsReady didn't work.

**Comment by 55** [comment]
```
Template.organizationCatalog.helpers({
  organizations () {
    // Return items of organization collection via Pagination
    return Template.instance().pagination.getPage();
  },
  ...
});
```
What is the reason to return items of organization collection via Pagination instead of collection helper or just query? For example, we do it this way to show "Nothing found." on the home page.

```
Template.latestPublicApis.helpers({
  latestPublicApis () {
    // Retrieve last API Backends
    return Apis.find({}, { sort: { created_at: -1 } }).fetch();
  },
});
```

**Comment by marla-singer** [comment]
> What is the reason to return items of organization collection via Pagination instead of collection helper or just query?
@55 APIs and Organizations catalogs have the pagination feature by [this package.](https://atmospherejs.com/kurounin/pagination) And the Doc of package dictates rules

**Comment by 55** [comment]
@marla-singer isn't enough just to check if there any APIs or Organizations exist to show these messages?

**Comment by marla-singer** [comment]
> isn't enough just to check if there any APIs or Organizations exist to show these messages?
@55 Maybe, actually, II don't understand enough about what exactly are you talking?

**Comment by brylie** [comment]
The problem is that the 'organization has no APIs' message flashes briefly while fetching data from the server. When the data is loading, we can perhaps show a spinner.

**Comment by brylie** [comment]
Temlplate helper to check if pagination subscription is ready:

```js
isReady: function () {
  return Template.instance().pagination.ready();
}
```

File: organizations/client/catalog/catalog.html [hunk 1]
```diff
@@ -16,21 +16,25 @@ <h1 class="page-header">
...
+          {{# if tableViewMode }}
+            {{> organizationCatalogTable organizations=organizations }}
+          {{/ if }}
+          {{> defaultBootstrapPaginator pagination=templatePagination limit=10 containerClass="text-center" }}
       {{ else }}
-        {{> spinner }}
+        <!-- Show no APIs message -->
+        {{> noOrganizationsMessage }}
+      {{/ if }}
+    {{/ if }}
```
Comment (brylie) [inline organizations/client/catalog/catalog.html]: Double check indentation here. There are several levels of nested statements, and the `{{ else }}` should be on the same level as its {{ if }}`.

**Comment by 55** [comment]
@brylie done.

File: organizations/client/catalog/catalog.html [hunk 2]
```diff
@@ -27,11 +27,11 @@ <h1 class="page-header">
             {{> organizationCatalogTable organizations=organizations }}
           {{/ if }}
           {{> defaultBootstrapPaginator pagination=templatePagination limit=10 containerClass="text-center" }}
-      {{ else }}
-        <!-- Show no APIs message -->
-        {{> noOrganizationsMessage }}
+        {{ else }}
+          <!-- Show no APIs message -->
```
Comment (brylie) [inline organizations/client/catalog/catalog.html]: Double check that comment matches code.

Comment (55) [inline organizations/client/catalog/catalog.html]: Good catch!

**Comment by 55** [comment]
@brylie will you merge it or there is anything for me to add?

**Pull request merged**